app/persistence/message/message.go

```diff
@@ -0,0 +1,37 @@
+package message
+
+import (
+	"gorm.io/gorm"
+)
+
+type TransactionMessage struct {
+	TransactionId string
+	EthAddress    string
+	Amount        uint64
+	Fee           string
+	Signature     string
+	Hash          string
+}
+
+type MessageRepository struct {
+	dbClient *gorm.DB
+}
+
+func NewMessageRepository(dbClient *gorm.DB) *MessageRepository {
+	return &MessageRepository{
+		dbClient: dbClient,
+	}
+}
+
+func (m MessageRepository) Get(txId, signature string) ([]TransactionMessage, error) {
+	var signatures []TransactionMessage
+	err := m.dbClient.Where("transaction_id = ? and signature = ?", txId, signature).Find(&signatures).Error
+	if err != nil {
+		return nil, err
+	}
+	return signatures, nil
+}
+
+func (m MessageRepository) Add(message *TransactionMessage) error {
```

Comment:
It would be good if all repository creation/addition methods have the same name.